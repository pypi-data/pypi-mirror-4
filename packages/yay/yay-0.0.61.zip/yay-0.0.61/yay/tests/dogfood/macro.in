
.define SomeMacro:
    - SomeItem:
        name: ${name}
    - SomeOtherItem:
        name: ${name}


resources.append:
    SomeMacro!:
        name: foo

resources.append:
    SomeMacro!:
        name: foobar

