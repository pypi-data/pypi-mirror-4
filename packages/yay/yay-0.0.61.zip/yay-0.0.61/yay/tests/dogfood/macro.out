
resources:
    - SomeItem:
        name: foo
    - SomeOtherItem:
        name: foo
    - SomeItem:
        name: foobar
    - SomeOtherItem:
        name: foobar

